extensions/single-file-stac/json-schema/schema.json

```diff
@@ -5,6 +5,12 @@
   "description": "Single File STAC Extension to combine Collections and Items in single file catalog",
   "allOf": [
     {
+<<<<<<< HEAD
+      "$ref": "../../catalog-spec/json-schema/catalog.json"
+    },
+    {
+=======
+>>>>>>> 99aa7da2760462aac1d304af8af5c303f9287173
```

Comment:
Looks like this is an unresolved merge conflict? 

Probably from my removal of item collection. I just threw in a feature collection reference, but I'm not sure if that's actually proper in json schema stuff.